doc/api.yml

```diff
@@ -2506,7 +2506,7 @@ paths:
             type: string
             format: date-time
         - name: commitment_end
-          required: true
+          required: false
```

Comment:
is it required or not ??

Reply:
They said that if the commitment_end is not provided and the commitment_begin is define then the student is allways WORKING